wg-charter.html

```diff
@@ -0,0 +1,364 @@
+      }
+
+    </style>
+  </head>
+  <body>
+    <header id="header">
+      <aside>
+        <ul id="navbar">
+          <li><a href="#scope">Scope</a></li>
+          <li><a href="#deliverables">Deliverables</a></li>
+          <li><a href="#coordination">Coordination</a></li>
+          <li><a href="#participation">Participation</a></li>
+          <li><a href="#communication">Communication</a></li>
+          <li><a href="#decisions">Decision Policy</a></li>
+          <li><a href="#patentpolicy">Patent Policy</a></li>
+          <li><a href="#licensing">Licensing</a></li>
+          <li><a href="#about">About this Charter</a></li>
+        </ul>
+      </aside>
+      <p>
+        <a href="https://www.w3.org/"><img alt="W3C" height="48" src="http://www.w3.org/Icons/w3c_home" width="72" /></a>
+
+      </p>
+    </header>
+
+    <main>
+      <h1 id="title">GPU for the Web Working Group Charter</h1>
+
+      <p class="mission">The <strong>mission</strong> of the <a
+      href="https://www.w3.org/gpuweb/">GPU for the Web Working Group</a> is to
+      provide an interface between the Web Platform and modern 3D graphics and
+      computation capabilities present on native system platforms.</p>
+
+      <div class="noprint">
+        <p class="join"><a href="https://www.w3.org/2004/01/pp-impl/FIXME/join">Join the GPU for the Web Working Group</a>.</p>
+      </div>
+
+      <section id="details">
+        <table class="summary-table">
+          <tr id="Duration">
+            <th>
+              Start date
+            </th>
+            <td>
+              TBD
+            </td>
+          </tr>
+          <tr id="Duration">
+            <th>
+              End date
+            </th>
+            <td>
+              TBD
+            </td>
+          </tr>
+
+          <tr>
+            <th>
+              Chairs
+            </th>
+            <td>
+              TBD
+            </td>
+          </tr>
+          <tr>
+            <th>
+              Team Contacts
+            </th>
+            <td>
+              TBD
+            </td>
+          </tr>
+          <tr>
+            <th>
+              Meeting Schedule
+            </th>
+            <td>
+              <strong>Teleconferences:</strong> as necessary.
+              <br />
+              <strong>Face-to-face:</strong> as necessary.
+            </td>
+          </tr>
+        </table>
+      </section>
+
+
+
+      <section id="scope" class="scope">
+        <h2>Scope</h2>
+        <p>
+            This Working Group will recommend a Web programming interface for
+            graphics and computation that:
+        </p>
+
+        <ol>
+            <li>enables rendering of modern graphics to both onscreen and offscreen drawing surfaces</li>
+            <li>enables computation tasks to be performed, and the results of such tasks to be retrieved</li>
+            <li>defines a Shading Language that describes the graphics and computation tasks in a format
```

Comment:
WHLSL is not part of the API itself and should not be tied to it since we have no consensus on using SPIRV or WHLSL as the ingestion format for WebGPU. All we agreed on is that it is a blessed language that could be defined in this group. (meaning we write some spec examples, tutorials and samples using it) 

I suggest "This group may also recommend a companion Shading Language that describes ..."

Reply:
✅